### PR TITLE
Stg78 ga prep

### PR DESCRIPTION
**Pull request by rickle-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 12.0.0-beta.9 (Unreleased)
+## 12.0.0-beta.9 (2021-09-15)
+### Dependency updates
```
Comment (g2vinay) [inline sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md:4]: Replace this with:
### Other Changes
#### Dependency Updates
- Updated blob dependency to 12.14.0

File: sdk/storage/azure-storage-blob-nio/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 12.0.0-beta.9 (Unreleased)
-
+## 12.0.0-beta.9 (2021-09-15)
+### Dependency Updates
```
Comment (g2vinay) [inline sdk/storage/azure-storage-blob-nio/CHANGELOG.md:4]: Replace this with:
### Other Changes
#### Dependency Updates
- Updated blob dependency to 12.14.0

File: sdk/storage/azure-storage-blob-batch/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.11.0-beta.2 (Unreleased)
-
+## 12.11.0 (2021-09-15)
+- GA release
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob-batch/CHANGELOG.md:4]: Maybe `This release contains bug fixes to improve quality.` like we do in .net when we absorb dependency updates. See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Common/CHANGELOG.md for example.

File: sdk/storage/azure-storage-internal-avro/CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 12.1.0-beta.2 (Unreleased)
-
+## 12.1.0 (2021-09-15)
+### Dependency Updates
```
Comment (g2vinay) [inline sdk/storage/azure-storage-internal-avro/CHANGELOG.md:4]: Replace this with:
### Other Changes
#### Dependency Updates
- Updated `azure-storage-common` to version `12.13.0`

**Review by g2vinay — APPROVED** [review]

File: sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md [hunk 5]
```diff
@@ -1,7 +1,9 @@
...
 
+- Updated blob dependency to 12.14.0
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md:6]: same here. Or maybe we just don't ship that one.

File: sdk/storage/azure-storage-blob-changefeed/README.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ process change events that occur in your Blob Storage account at a low cost.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-storage-blob-changefeed</artifactId>
-    <version>12.0.0-beta.7</version>
+    <version>12.0.0-beta.9</version>
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob-changefeed/README.md:23]: why is there a jump from 7 to 9 ?

File: sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.14.0-beta.2 (Unreleased)
-
+## 12.14.0 (2021-09-15)
+- GA release
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md:4]: same here.

File: sdk/storage/azure-storage-blob-nio/README.md [hunk 8]
```diff
@@ -19,7 +19,7 @@ This package allows you to interact with Azure Blob Storage through the standard
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-storage-blob-nio</artifactId>
-    <version>12.0.0-beta.7</version>
+    <version>12.0.0-beta.9</version>
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob-nio/README.md:22]: similar comments like for changefeed.

File: sdk/storage/azure-storage-blob/CHANGELOG.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.14.0-beta.2 (Unreleased)
-
+## 12.14.0 (2021-09-15)
+- GA release
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/CHANGELOG.md:4]: might be worth mentioning that this includes all features from beta. See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/CHANGELOG.md#12100-2021-09-08

File: sdk/storage/azure-storage-blob-changefeed/README.md (hunk 6 above)
Comment (rickle-msft) [reply sdk/storage/azure-storage-blob-changefeed/README.md:23]: I think the readme was just not updated in the last release. [beta8](https://search.maven.org/artifact/com.azure/azure-storage-blob-changefeed/12.0.0-beta.8/jar) has already been shipped

File: sdk/storage/azure-storage-blob-nio/README.md (hunk 8 above)
Comment (rickle-msft) [reply sdk/storage/azure-storage-blob-nio/README.md:22]: confirmed [beta8](https://search.maven.org/artifact/com.azure/azure-storage-blob-nio/12.0.0-beta.8/jar) was also released here. I think this was also a readme oversight.

File: sdk/storage/azure-storage-blob-batch/CHANGELOG.md (hunk 3 above)
Comment (rickle-msft) [reply sdk/storage/azure-storage-blob-batch/CHANGELOG.md:4]: Does it contain bug fixes to improve quality? It's just a release to move features previously released as beta to stable

File: sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md (hunk 5 above)
Comment (rickle-msft) [reply sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md:6]: I think we should continue to release everything in step to make dependency graphs and boms easier

File: sdk/storage/azure-storage-blob-batch/CHANGELOG.md (hunk 3 above)
Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob-batch/CHANGELOG.md:4]: through dependency update of core and blobs perhaps. Not many features in beta.

File: sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md (hunk 5 above)
Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md:6]: for GA packages yes, but I'm not sure if it makes sense to ship any new betas if we didn't add anything.

**Review by kasobol-msft — APPROVED** [review]

**Pull request merged**